src/rdev.rs

```diff
@@ -25,19 +26,35 @@ pub enum ListenError {
     RecordContextError,
     /// Linux
     XRecordExtensionError,
+    /// Windows
+    KeyHookError(u32),
+    /// Windows
+    MouseHookError(u32),
 }
 
 /// Errors that occur when trying to grab OS events.
 /// Be careful on Mac, not setting accessibility does not cause an error
 /// it justs ignores events.
 #[derive(Debug)]
+#[non_exhaustive]
 pub enum GrabError {
     /// MacOS
     EventTapError,
     /// MacOS
     LoopSourceError,
     /// Linux
     LinuxNotSupported,
+    /// Windows
+    KeyHookError(u32),
+    /// Windows
+    MouseHookError(u32),
+}
+/// Errors that occur when trying to get display size.
+#[non_exhaustive]
+#[derive(Debug)]
+pub enum DisplayError {
```

Comment:
can you pause adding error types for a bit? I think this crate should only have one unified error type that includes all these - they're not that different, and have a fair amount of overlap.

Reply:
I agree with you. I'll try to rework errors.
Not sure it belongs in this PR though. 

I prefer to have small PRs with unitary logic if possible.

Reply:
> Not sure it belongs in this PR though

I agree that it doesn't belong in this PR, but it is hard to mess with error code while this PR is open (merge conflicts and such). So I suppose it's best to merge it quickly, and at the next opportunity, write a unified error type.